packages/orbit-components/src/tmp-tailwind/InputField/README.md

```diff
@@ -0,0 +1,138 @@
+# InputField
```

Comment:
pls remove redundant files from `tmp-tailwind`, keep only the story and the main component file 🙏

Reply:
I do not think it's a good idea remove __tests__ folder, I can remove the types and README